apps/docs/content/guides/local-development/testing/pgtap-extended.mdx

```diff
@@ -345,17 +341,30 @@ create table public.comments (
 
 #### 2. RLS policies declaration
 
-Now we want to setup the RLS policies for each of our tables:
+Now to setup the RLS policies for each tables:
 
 ```sql
+-- Create a private schema to store all security definer functions utils
+-- As such functions should never be in a API exposed schema
+create schema if not exists private;
 -- Helper function for role checks
-create or replace function public.get_user_org_role(org_id bigint, user_id uuid)
+create or replace function private.get_user_org_role(org_id bigint, user_id uuid)
```

Comment:
```suggestion
create or replace function private.get_user_org_role(org_id bigint, user_id uuid)
set search_path = ''
```

forgot this last time, otherwise it will trigger database linting errors